dash/testing/browser.py

```diff
@@ -148,7 +153,43 @@ def percy_snapshot(self, name="", wait_for_callbacks=False):
                 self.redux_state_rqs,
             )
 
-        self.percy_runner.snapshot(name=snapshot_name)
+        if convert_canvases:
+            self.driver.execute_script(
+                """
+                const stash = window._canvasStash = [];
+                Array.from(document.querySelectorAll('canvas')).forEach(c => {
+                    const i = document.createElement('img');
+                    i.src = c.toDataURL();
+                    i.width = c.width;
+                    i.height = c.height;
+                    i.setAttribute('style', c.getAttribute('style'));
+                    i.className = c.className;
+                    i.setAttribute('data-canvasnum', stash.length);
+                    stash.push(c);
+                    c.parentElement.insertBefore(i, c);
+                    c.parentElement.removeChild(c);
+                });
+            """
+            )
```

Comment:
New feature for `dash.testing` - `percy_snapshot` adds a kwarg `convert_canvases` that turns every `<canvas>` into an `<img>` with matching contents, then puts the canvases back afterward. This lets us capture Cytoscape, among other things (cc @xhlulu)
<img width="665" alt="Screen Shot 2020-08-29 at 11 39 51 PM" src="https://user-images.githubusercontent.com/2678795/91650758-666ea800-ea51-11ea-99aa-563ccf65ecda.png">
Sizing isn't quite as expected, but I assume that's just Percy rendering as a different screen size from what's used in circleci.